mrjob/spark/mrjob_spark_harness.py

```diff
@@ -47,9 +50,31 @@
         dest='compression_codec',
         help=('Java class path of a codec to use to compress output.'),
     )),
+    (['--counter-output-dir'], dict(
+        default=None,
+        dest='counter_output_path',
```

Comment:
`dest` should match the switch (it should be `counter_output_dir`)

(argparse would also do this automatically if you didn't set `dest`, but In `mrjob` I always set the `dest` explicitly because I like to be able to grep for things.)

Reply:
ah I rename it at some point, and forget to change everything. Thanks for finding it